inductiva/fluids/scenarios/coastal_area/bathymetry.py

```diff
@@ -253,16 +266,43 @@ def plot(
               the matplotlib `Axes` object is returned instead.
         """
 
+        # Create uniform grid for interpolation.
+        x_grid, y_grid = np.meshgrid(np.linspace(*self.x_range, grid_size),
+                                     np.linspace(*self.y_range, grid_size))
+
+        # Interpolate depths to uniform grid.
```

Comment:
I am not sure,  if this will be the right place to perform this. Given that this will be used by the simulator as well. Wdyt?

Reply:
I agree that it doesn't feel right to have the interpolation code here. I am still figuring out the best place to put this... maybe under some interpolation utilities?

Reply:
Right, but this plotting utility is meant to be used for arbitrary bathymetries, even those that can't be used to run SWASH simulations. The goal of the utility is exactly to visualize the bathymetry, so the user can inspect which region to crop and simulate.

Reply:
Yeah, I see that point. If needed we can adapt later!